self_enrollment_tool/templates/self_enrollment_tool/index.html

```diff
@@ -89,9 +89,11 @@ <h4 class="modal-title bottom-spacing">Self Enrollment Course List:</h4>
                         </td>
                         {% comment %} Self Enroll URL column {% endcomment %}
                         <td>
-                            <a href="{{ crs.self_enroll_url }}">
-                                {{ crs.self_enroll_url }}
-                            </a>
+                            <div>
+                                <span id="url">{{ crs.self_enroll_url }}</span>
+                                <br>
+                                <button id="copy" class="btn btn-primary" onclick="copy(event)">Copy URL</button>
```

Comment:
I think the button is a bit large; we could use an icon instead to make it fit a little better. 

Since we have FontAwesome in our template you can display an icon like this:
```
<i class="fa fa-clone"></i>
```

Reply:
Sounds good! Will update.